utility_analysis/peeker_engine.py

```diff
@@ -159,9 +159,9 @@ def _partition_selection_filter_fn(
 ) -> bool:
     """Lazily creates a partition selection strategy and uses it to determine which
     partitions to keep."""
-    privacy_id_count, _ = row[1]
+    pirvacy_id_count, _ = row[1]
     partition_selection_strategy = (
         pydp.algorithms.partition_selection.
         create_truncated_geometric_partition_strategy(budget.eps, budget.delta,
                                                       max_partitions))
-    return partition_selection_strategy.should_keep(privacy_id_count)
+    return partition_selection_strategy.should_keep(pirvacy_id_count)
```

Comment:
This returns back previous typo and in other places. The easies way to fix is merge main to this branch